packages/web/src/shared/ChangeStory.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { useScreenSize } from 'src/layout/ScreenSize'
+import globe from 'src/shared/world-spin.gif'
+import { colors, fonts, standardStyles, textStyles } from 'src/styles'
+
+const CHANGE_STORY = [
+  'Change the Story', // en
+  'Cambia la historia', // es
+  '改变故事', // cn
+  'Muda a história', // pt
+
+  // '', // ko
```

Comment:
Was keeping this intentional?

Reply:
a reminded that korean translation should drop any day now